src/util/chaintype.h

```diff
@@ -16,6 +16,8 @@ enum class ChainType {
     TESTNET4,
 };
 
+std::string ChainTypeToDisplayString(ChainType chain);
+
 std::string ChainTypeToString(ChainType chain);
```

Comment:
The header order should ideally be the same as the implementation, unless there's a reason for it:
```suggestion
std::string ChainTypeToString(ChainType chain);
std::string ChainTypeToDisplayString(ChainType chain);
```